_rules/SC1-3-5-autocomplete-valid.md

```diff
@@ -165,76 +190,92 @@ Autocomplete is inappropriate for the type of field.
 <input type="number" autocomplete="email" />
 ```
 
+#### Failed example 6
+
+Autocomplete is not empty, but does not have any terms specified.
+
+```html
+<input autocomplete=" " />
```

Comment:
I don't think this should fail. I think you should put back the "not only whitespace" thing in this rule.

Reply:
Why should the above not fail if something like the following does?

```html
<input autocomplete="." />
```

Both were obviously attempts at giving the attribute a non-zero-length value.

Reply:
@WilcoFiers, @kasperisager and @annethyme agreed on a call to try to go with the current wording, since it's not all types of whitespace that will be trimmed in attributes like this, and differentiating between the different types of whitespace will complicate the rule, which we don't really know is necessary. 
However, if we, when running the rule in real life settings, see a lot of cases where we are catching spaces and control characters, that are not really causing any accessibility issues, we might have to re-evaluate this decision.